### DISCUSSION TITLE
Severe Visual Interference

### DISCUSSION THREAD

**Discussion by CodingParamedic** [atom/atom#22402] [Q&A]
Hey folks,

I've been getting severe visual disturbances (pic attached) when using Atom for any amount of time. The picture is one of the more severe moments. I can clear it by minimizing and then re-opening it, or clicking on the area with the interference.

Has anyone else experienced this? It seems to get worse the longer i use it, or when it's working hard on something. For instance, when installing plug-ins, it got so bad i couldn't read or understand anything on the screen.

This has happened with both plug ins installed, and without.

Thanks for your time.

![issues](https://user-images.githubusercontent.com/83573177/117924980-39e77d00-b2bc-11eb-9477-0d776f54889b.jpg)

**Comment by Saptarshi-Daw** [comment]
hey, I am facing the same disturbance I can't figure out what's going wrong.

**Comment by icecream17** [comment]
> Are you able to inspect it with devtools?
> 
> 1. <kbd>Ctrl</kbd> + <kbd>Shift</kbd> + <kbd>i</kbd>
> 2.  Click on the arrow in the top left.
> ![image](https://user-images.githubusercontent.com/97994/114075693-706c4b00-986b-11eb-981a-f1f15bfad379.png)
> 3. Click on the object.
> 
> _Originally posted by U ziTech in https://github.com/atom/atom/discussions/22147#discussioncomment-586150_

This is... very interesting to say the least. It reminds me of that stackoverflow answer about regex and html.

**Reply by icecream17** [reply]
Oh, I found something: https://www.reddit.com/r/vscode/comments/ep1t8d/what_is_going_on_here_accidentally_creating_a/fegknjg?utm_source=share&utm_medium=web2x&context=3

> Can you turn off hardware acceleration per program? I haven't run graphics in a few years, but that could be a combination of anti-aliased fonts in VS being cyclically anti-aliased by a graphics card?
> 
> Shooting in the dark here, someone tell me why I'm an idiot.

Maybe try this @CodingParamedic @Saptarshi-Daw
